oauth2_http/java/com/google/auth/oauth2/IdentityPoolCredentialSource.java

```diff
@@ -45,9 +47,193 @@ public class IdentityPoolCredentialSource extends ExternalAccountCredentials.Cre
   private static final long serialVersionUID = -745855247050085694L;
   IdentityPoolCredentialSourceType credentialSourceType;
   CredentialFormatType credentialFormatType;
-  String credentialLocation;
+  private String credentialLocation;
```

Comment:
Made `credentialLocation` and `certificateConfig` fields private and added public getters to improve encapsulation and provide the ability to properly mock the object.